concordium-consensus/src/Concordium/GlobalState/Persistent/BlockState.hs

```diff
@@ -1783,7 +1783,13 @@ updateValidatorChecks bsp baker ValidatorUpdate{..} = do
 --
 --        (3) append @BakerConfigureFinalizationRewardCommission frc@ to @events@.
 --
---  8. If the capital is supplied: if there is a pending change to the baker's capital, return
+--  8. (>= P8) If the suspended/resumed flag is set:
+
+--        (1) Suspend/resume the validator according to the flag.
+
+--        (2) Append @BakerConfigureSuspended@ or @BakerConfigureResumed@ accordingly to @events@.
+--
```

Comment:
This should also be reflected in the docs on `bsoUpdateValidator`.

Reply:
done.